Software/grab/GrabberBase.cpp

```diff
@@ -174,7 +174,11 @@ void GrabberBase::grab()
             const int bytesPerPixel = 4;
             QRgb avgColor;
             if (_context->grabWidgets->at(i)->isAreaEnabled()) {
-                Calculations::calculateAvgColor(&avgColor, grabbedScreen->imgData, grabbedScreen->imgFormat, grabbedScreen->screenInfo.rect.width() * bytesPerPixel, preparedRect );
+                Q_ASSERT(grabbedScreen->imgData);
+                Calculations::calculateAvgColor(
+                    &avgColor, grabbedScreen->imgData, grabbedScreen->imgFormat,
+                    grabbedScreen->screenInfo.rect.width() * bytesPerPixel,
+                    preparedRect );
```

Comment:
I know formatting is far from consistent, but if you change it, please remove the space before the closing bracket. I prefer having one line per argument (if multi line) for readability.